parquery/aggregate.py

```diff
@@ -37,15 +37,16 @@ def aggregate_pq(
 
     # if the file does not exist, give back an empty result
     if not os.path.exists(file_name) and handle_missing_file:
-        df = pd.DataFrame(None, columns=result_cols)
-        if as_df:
-            return df
-        else:
-            return pa.Table.from_pandas(df, preserve_index=False)
+        return create_empty_result(result_cols, as_df)
 
     # get result
     pq_file = pq.ParquetFile(file_name)
 
+    # check if we have all dimensions from the filters
+    for col, _, _ in data_filter:
+        if col not in pq_file.metadata.schema.names:
+            return create_empty_result(result_cols, as_df)
```

Comment:
This loops through all `data_filter` and if the col is not in `metadata.schema` it will return an empty result? Even if there are filters that they are in  `metadata.schema`? Shouldnt we just fill somehow the missing ones? Instead of giving back an empty dataframe? Is this the reason of the failing test?

Reply:
Hi! if you filter on something that is not in available in the parquet file, we really should give an empty result